examples/core-ping.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://mozilla.org/schemas/glean/ping",
+  "ping_info": {
+    "ping_type": "core",
+    "app_build": "59f330e5...",
+    "telemetry_sdk_build": "abcdabcd...",
+    "client_id": "f99fbfb6...",
```

Comment:
nit: let's have a proper UUID here, since this is an example, something like `36f7deeb-9f16-424f-a234-d0021e494ef7`. Let's do the same for other examples as well: let's keep samples as close to reality as possible